README.md

```diff
@@ -0,0 +1,76 @@
+# Setup Guide  
```

Comment:
Good idea with introducing README.md with setup nodes. May I ask you please also include Contentful arguments declaration?

Reply:
Sure, it makes sense, will do